recipes/klayout/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = "0.27.12" %}
+
+package:
+  name: klayout
+  version: {{ version }}
+
+source:
+  url: https://github.com/KLayout/klayout/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6ef5e6ae5ea387618663949a1efa105406aeb36db2d18d89eff0345955aae979
+  patches:
+    - force-python-dynamic-link.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  ignore_run_exports:
+    - qt
+  missing_dso_whitelist:  
+    - "*/libnet_tracer.so.0"
+    - "*/liblefdef.so.0"
+    - "*/libpcb.so.0"
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ compiler('c') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}  # [linux]
+    - {{ cdt('libxfixes') }}  # [linux]
+    - {{ cdt('libxau') }}  # [linux]
+    - {{ cdt('libxcb') }}  # [linux]
+    - {{ cdt('expat') }}  # [linux]
+    - {{ cdt('libpng') }}  # [linux]
+
+  host:
+    - ruby >2
+    - qt >=5.0,<6.0
+  run:
+    - qt-main
+    - python
+    - libzlib
+    - libpython-static
```

Comment:
```suggestion
```
The run section is for dependencies which are needed at runtime, but aren't needed during any compilation stage. This includes packages which provide binaries that are just being used for their CLIs (curl) and most python packages.

There's no point in having libpython-static as a runtime dependency. You can't use a static library file without a compiler?